torch/library.py

```diff
@@ -71,13 +71,17 @@ def __init__(self, ns, kind, dispatch_key=""):
     def __repr__(self):
         return f"Library(kind={self.kind}, ns={self.ns}, dispatch_key={self.dispatch_key})>"
 
-    def define(self, schema, alias_analysis=""):
+    def define(self, schema, alias_analysis="", *, tags=()):
         r'''Defines a new operator and its semantics in the ns namespace.
 
         Args:
             schema: function schema to define a new operator.
             alias_analysis (optional): Indicates if the aliasing properties of the operator arguments can be
                                        inferred from the schema (default behavior) or not ("CONSERVATIVE").
+            tags (optional): Tuple of torch.Tag to add to the operator. Adding
+                             torch.Tag may change the behavior of the operator,
+                             please throughly test the operator to ensure the
+                             Tag applies.
```

Comment:
A typical end user isn't going to know what "test the operator so the tag applies" means.

Reply:
Thanks for the feedback, I'll link the gdoc.